packages/edit-widgets/src/components/widget-areas-block-editor-provider/index.js

```diff
@@ -74,6 +56,12 @@ export default function WidgetAreasBlockEditorProvider( {
 		};
 	}, [ blockEditorSettings, hasUploadPermissions ] );
 
+	const [ blocks, onInput, onChange ] = useEntityBlockEditor(
+		KIND,
+		POST_TYPE,
+		{ id: buildWidgetAreasPostId() }
```

Comment:
What is the reason that makes need to have a post id and use useEntityBlockEditor here? Using it inside the widget area block cause some problem?